### PR TITLE
Enable to play sound

### PR DESCRIPTION
**Pull request by mehanoid**
Based on https://github.com/marzocchi/zsh-notify/pull/28, added support for Linux.

### CODE REVIEW

File: notify-if-background [hunk 1]
```diff
@@ -166,6 +172,15 @@
         function notifier-linux {
             notify-send "$notification_title" "$message"
 
+            if [[ ! -z "$notification_sound" ]]; then
+                if [[ -a $notification_sound ]]; then
+                    local player=`(which paplay || which aplay || which ossplay) | tail -n 1`
+                    eval $player $notification_sound
+                else
+                    echo "zsh-notify: file $notification_sound not found."
```
Comment (marzocchi) [inline notify-if-background]: please send to stderr [like in here](https://github.com/marzocchi/zsh-notify/blob/master/notify-if-background#L187)

**Comment by marzocchi** [comment]
nice one, thanks!

**Comment by marzocchi** [comment]
@mehanoid so there's now a new PR #30 in which I extracted some code to a function, can you have a look at it?

**Comment by mehanoid** [comment]
@marzocchi, tried out https://github.com/marzocchi/zsh-notify/pull/30, everything is ok!

**Comment by marzocchi** [comment]
thank you @mehanoid

**Pull request merged**